command/agent/config.go

```diff
@@ -516,6 +516,10 @@ type ServerConfig struct {
 	ExtraKeysHCL []string `hcl:",unusedKeys" json:"-"`
 
 	Search *Search `hcl:"search"`
+
+	// DeploymentRateLimit is in queries per second and is used by the
+	// DeploymentWatcher to throttle the amount of simultaneously deployments
+	DeploymentRateLimit float64
```

Comment:
left the variable as a float64 bc that's what the `rate` package accepts. realistically though we probably wouldn't want a user to specify a fractional value?

Reply:
Ah sure. `float64` is fine. I think simplicity of code outweighs ergonomics of use for really rarely used parameters like this.

If we find a new value that's a better default, we should just switch defaults instead of encouraging people to think about this parameter.